src/vs/workbench/contrib/webview/browser/pre/index-no-csp.html

```diff
@@ -701,6 +701,24 @@
 			return hasMeta && e.keyCode === 78;
 		}
 
+		/**
+		 * @param {KeyboardEvent} e
+		 * @return {boolean}
+		 */
+		function isHelp(e) {
+			// 112: keyCode of "F1"
+			return e.keyCode === 112;
+		}
+
+		/**
+		 * @param {KeyboardEvent} e
+		 * @return {boolean}
+		 */
+		function isRefresh(e) {
+			// 116: keyCode of "F5"
+			return e.keyCode === 116;
```

Comment:
Isn't the keyCode property being [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode), in favor of using the `KeyboardEvent.code` property?